blog/content/post/2024-11-04-qubes-os-summit-post.md

```diff
@@ -0,0 +1,226 @@
+
+She discussed integrating modern design elements to align Qubes OS with
+contemporary usability standards and improvements in the new devices widget,
+such as automatically attaching the new device to a qube.
+
+After the short break, Piotr started the design session:
+[Enhancing OS Awarness of Hardware Security Capabilities in Qubes OS](https://www.youtube.com/watch?v=tT9ss8gQYm8&t=5s).
+He highlighted how Qubes is evolving to integrate better and leverage hardware
+security features to enhance system security. During the session, a fascinating
+discussion arose among the conference participants, including some interfaces
+for CPU assessment.
+
+Nestire presented the talk
+[Passwordless encrypted Qubes? Exploring some concepts](https://www.youtube.com/watch?v=GUOnBapSLRE&t=5s).
+In his talk, he focused on possible ways to prevent attacks. This approach aims
+to enhance user experience without compromising security by using
+alternative, hardware-backed authentication methods in place of traditional
+passwords.
+
+At the Qubes OS Summit 2024, the talk on
+[How to architect your Qubes OS with SaltStack](https://www.youtube.com/watch?v=GUOnBapSLRE&t=5s)
+hosted by Benjamin Grande emphasized how SaltStack can be leveraged to automate
+and manage Qubes OS configurations efficiently.
+
+The following presentation was
+[FlashKeeper: where SpiSpy meets Stateless Laptop jaded dreams: A retrofit plan first](https://www.youtube.com/watch?v=DxFceGi6C0k)
+held by Thierry Laurion. Thierry pointed out the advantages of using FlashKeeper
+because of its quick flashing cycles. He pointed out that for users concerned
+with physical attacks on their systems, for whom easy access to SPI flash pins
+may be seen as a risk, a variant including a small FPGA closely collocated with
+the flash is also being developed.
+
+The presentation delves into the convergence of two key projects: SpiSpy, a tool
+for monitoring SPI flash chips, and efforts toward stateless laptops, which
+eliminate sensitive data persistence. The talk outlined a retrofit plan to
+enhance hardware security, particularly within the Qubes OS ecosystem.
+
+Michał Żygowski talked about [Anti Evil Maid status and future plans](https://www.youtube.com/watch?v=5ieNhbLLTIU).
+AEM is a security tool designed to protect against attacks that tamper with the
+boot process of systems like Qubes OS. Michał presented an interesting case on
+the performance of AEM on NovaCustom NV4x Alderlake laptop and the results he
+achieved.
+
+Marek Marczykowski-Górecki and Frédéric Pierret presented an [Update on Qubes Air](https://www.youtube.com/watch?v=V4flhwEITr4),
+indicating the current issues, for example focus on Qrexec calls to other
+systems anddisposable qube support.
+
+Moreover, in this version, there is no GUI support for remote qubes, focus is on
+individual VMS on both systems and not on automatically synchronizing all the
+cube machines to the other. There is also a new type of Qube – a Relay Qube – a
```

Comment:
we have to decide either we use Qube or qube, post should be consistent

Reply:
@gosiastyba @MartaWitt this is not fixed. Previous paragraph use `qube` here we use `Qube`

Reply:
The issue has been corrected.